recipes/hole2/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "hole2" %}
+{% set version = "2.3.2" %}
```

Comment:
any chance we can set this back to 2.3.0 to match the planned 2.3.0 release https://github.com/osmart/hole2/issues/5 ?

Reply:
yeah that's the plan for right now